include/GafferArnoldUI/Export.h

```diff
@@ -0,0 +1,46 @@
+//////////////////////////////////////////////////////////////////////////
+//
+//  Copyright (c) 2020, Cinesite VFX Ltd. All rights reserved.
+//
+//  Redistribution and use in source and binary forms, with or without
+//  modification, are permitted provided that the following conditions are
+//  met:
+//
+//     * Redistributions of source code must retain the above copyright
+//       notice, this list of conditions and the following disclaimer.
+//
+//     * Redistributions in binary form must reproduce the above copyright
+//       notice, this list of conditions and the following disclaimer in the
+//       documentation and/or other materials provided with the distribution.
+//
+//     * Neither the name of Cinesite VFX Ltd. nor the names of any
+//       other contributors to this software may be used to endorse or
+//       promote products derived from this software without specific prior
+//       written permission.
+//
+//  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS
+//  IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO,
+//  THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+//  PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+//  CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+//  EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+//  PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+//  PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+//  LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+//  NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+//  SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+//
+//////////////////////////////////////////////////////////////////////////
+
+#ifndef GAFFERARNOLDUI_EXPORT_H
+#define GAFFERARNOLDUI_EXPORT_H
+
+#include "IECore/Export.h"
+
+#ifdef GafferArnold_EXPORTS
```

Comment:
Was just about to merge but spotted this at the last minute - shouldn't this be `#ifdef GafferArnoldUI_EXPORTS`, and if so, how has it been working at all so far?

Reply:
Good spot - thanks. It totally should. I think it's because on linux/mac it doesn't make any difference? Its only on needed Windows for `<fun window reasons>`, see [here](https://github.com/ImageEngine/cortex/blob/821bfb536d6b2e16a829930e27a7f83bf604bce6/include/IECore/Export.h#L40).  Fixed in https://github.com/GafferHQ/gaffer/pull/3667/commits/5754baa261814b8617d79d436ce162f43fa15ca9.

Reply:
Ah, that makes sense. Thanks for the fix - will merge once CI is done.